west.yml

```diff
@@ -32,6 +32,9 @@ manifest:
       revision: 908ffde6298a937c6549dbfa843a62caab26bfc5
       import:
         path-prefix: tools
+    - name: acpica
```

Comment:
It looks like all modules are in alphabetical order, so you should move this above bsim